lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1889,7 +1879,16 @@ void ClangImporter::Implementation::handleDeferredImports(SourceLoc diagLoc) {
   // officially supported with bridging headers: app targets and unit tests
   // only. Unfortunately that's not enforced.
   for (size_t i = 0; i < ImportedHeaderExports.size(); ++i) {
```

Comment:
Nothing you changed, but I was wondering why not use a `for( auto *CM: ImportedHeaderExports)` loop?

Reply:
Unfortunately, `ImportedHeaderExports` may have elements pushed into it by module loading while it is running. I haven't verified if this re-entrancy no longer occurs - I haven't even found a test that exercises the current code path.